torch/linalg/__init__.py

```diff
@@ -209,21 +221,20 @@
 """)
 
 slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
-linalg.slogdet(input, *, out=None) -> (Tensor, Tensor)
+linalg.slogdet(A, *, out=None) -> (Tensor, Tensor)
 
-Computes the sign and logarithm of the norm of the determinant of a square matrix. Supports batched inputs.
+Computes the sign and natural logarithm of the absolute value of the determinant of a square matrix.
 
-For a complex :attr:`input`, it returns the angle and the logarithm of the modulus of the
-determinant, that is, a logarithmic polar decomposition of the determinant.
 
-It returns a named tuple `(sign, logabsdet)`.
-If :attr:`input` is a batch of matrices, then `sign`, `logabsdet` are also batched with the same
-batch dimensions.
+For a complex :attr:`A`, it returns the angle and the natural logarithm of the modulus of the
+determinant, that is, a logarithmic polar decomposition of the determinant.
 
-Supports :attr:`input` of float, double, cfloat and cdouble dtypes.
-The output tensor `logabsdet` will always be real-valued, even when :attr:`input` is complex.
-The output tensor `sign` will have the same dtype as :attr:`input`.
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
 
+It returns a named tuple `(sign, logabsdet)`.
```

Comment:
Should this be in a special "Returns" section? We're not consistent about having that section in the docs for every operator, but when an operator returns a named tuple that's typically where we elaborate on it

Reply:
This is a very good idea. The only one in which I am tempted to leave outside of `Return` is `lstsq`, as it currently has that crazy 4-tuple of a return. If put in the `Return` section, the list looks really crammed.

Reply:
In fact, I'm going to use that section to talk about the type of the returned tensor, whenever there is something to say about it (e.g. norm is always real even when the input tensor is complex).